src/gui/rss/automatedrssdownloader.ui

```diff
@@ -96,6 +96,16 @@
            </widget>
           </item>
           <item>
+           <widget class="QCheckBox" name="checkSmart">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Smart Episode Filter will checks the episode number to prevent it downloading duplicates. Supports the formats: S01E01, 1x1 and 2017.01.01&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Also here is a better string with no grammatical or syntax errors.
```
Smart Episode Filter will check the episode number to prevent downloading of duplicates. Supports the formats: S01E01, 1x1 and 2017.01.01
```